### PR TITLE
LIS: add focus problem

### PR DESCRIPTION
**Pull request by frodakcin**
https://github.com/cpinitiative/usaco-guide/issues/839

### CODE REVIEW

**Review by dongliu0426 — APPROVED** [review]

File: content/4_Gold/LIS.mdx [hunk 1]
```diff
@@ -353,6 +362,8 @@ class PCB{
 
 ## Extension - LCS on Permutations
 
+<FocusProblem problem={problems.lisperm} />
+
 Given two permutations of length $N$ ($1\le N \le 10^5$), find the longest common subsequence.
 
 ### Example
```
Comment (dongliu0426) [inline content/4_Gold/LIS.mdx:369]: actually, could you remove the sample input explanation here and add it to the statement https://codeforces.com/gym/102951/problem/C?

**Comment by frodakcin** [comment]
Ok, it just updated on cf

**Comment by frodakcin** [comment]
Think it's good to merge?

**Comment by dongliu0426** [comment]
> Think it's good to merge?

yes

**Pull request merged**